### ISSUE TITLE
Unable to tab complete included config files 

### ISSUE THREAD

**Issue by ryan-cpi** [scop/bash-completion#656]
## Describe the bug

Tab complete not working for includes in ssh config file. 

## Install 

bash-completion was installed with `brew install bash-completion` 
`brew install bash-completion@2` wont work as macs only come with bash 3.2 due to licensing 

## To reproduce

SSH config file
```
~/.ssh$ cat config
# Sandbox
Host test
    user ubuntu
    port 22
    hostname 8.8.8.8

Include ~/.ssh/sandbox_config
```
Include file 
```
:~/.ssh$ cat sandbox_config 
Host sandbox-server
    HostName localhost
    User ubuntu
    IdentitiesOnly yes
    Port 10098
```

bash_profile 
```
tail ~/.bash_profile 
# Bash autocompletion
[[ -r "$(brew --prefix)/etc/profile.d/bash_completion.sh" ]] && . "$(brew --prefix)/etc/profile.d/bash_completion.sh"
```

## Expected behavior

Able to tab complete for `sandbox-server` however im only able to tab for `test`

## Versions 

- Im running macOS Monterey version 12.0.1
- echo "$BASH_VERSION" **3.2.57(1)-release**
- [**null**] bash-completion version, `(IFS=.; echo "${BASH_COMPLETION_VERSINFO[*]}")`:

**Comment by akinomyoga** [comment]
This also reproduces with the current `master` branch. Just `Include` directives are not supported by `_known_hosts_real` (`bash_completion`).

----

> SSH config file
> 
> ```console
> ~/.ssh$ cat config
> # Sandbox
> Host test
>     user ubuntu
>     port 22
>     hostname 8.8.8.8
> 
> Include ~/.ssh/sandbox_config
> ```

This is unrelated to the issue here, but I doubt that your ssh configuration works. Have you ever executed `ssh sandbox-server` with this setting? `Include` directives for host settings need to be specified before any `Host` or `Match` sections, i.e.,

```diff
 # Sandbox
+Include ~/.ssh/sandbox_config

 Host test
     user ubuntu
     port 22
     hostname 8.8.8.8

-Include ~/.ssh/sandbox_config
```

**Comment by ryan-cpi** [comment]
Thanks for the fast reply. That config was something i quickly put together as an example for the tab completion. It gives the same result wherever its placed but good spot.

This issue has been reported in the past as theres previous commits https://github.com/scop/bash-completion/commit/1016e4336c2eaa95888275df6ea248b4e913fb76

**Comment by akinomyoga** [comment]
> This issue has been reported in the past as theres previous commits [1016e43](https://github.com/scop/bash-completion/commit/1016e4336c2eaa95888275df6ea248b4e913fb76)

Ah, OK. I now again tried and it actually works with the current `master` branch.

Since it is already fixed in the `master` branch, what we can do is to backport the support for the `include` directive to the version which is used with bash-3.2 in macOS. #345 is the related issue. It seems currently there are no maintainers for bash-completion-1.x.

**Comment by akinomyoga** [comment]
I don't think this issue is soon resolved with Bash 3.2. Also, there are many other improvements in bash-completion, so I suggest installing the latest version of Bash in macOS by yourself through `brew` or using the manual installation.

**Comment by ryan-cpi** [comment]
Appreciate the fast response bash 5.1 works fine on mac when all setup correctly
